### DISCUSSION TITLE
DOMNodeInserted mutation event type is deprecated, and will be removed from this browser very soon. Usage of this event listener will cause performance issues today, and represents a large risk of future site breakage.

### DISCUSSION THREAD

**Discussion by GediminasKilt** [vueup/vue-quill#549] [General]
I'm using vue-quill package and I get this warning in console. Could you please fix this issue sooner then event is removed from browser which might include breakage of vue-quill package. 
I',m adding error print screen and code location I think is causing this warning:
![Screenshot 2024-05-20 at 17 32 44](https://github.com/vueup/vue-quill/assets/135216251/23277deb-2b3f-4f20-adef-5d31f2d6616e)
![Screenshot 2024-05-20 at 17 32 53](https://github.com/vueup/vue-quill/assets/135216251/1fccaca3-c64c-408a-aab7-5dcc4c6e8b02)

**Comment by al1maher** [comment]
Did you find a solution?
